src/css/result.css

```diff
@@ -78,3 +78,16 @@ body {
 .Main___StyledScrollView-sc-1n64cjc-0 .hZYLiF {
   height: auto;
 }
+
+.MenuPanel___StyledDiv2-sc-1warxvz-4 .egeaz {
```

Comment:
Was this change to the Menu panel intentional? This removes the boldness of the menu items right?